wiki/Welcome/id.md

```diff
@@ -1,136 +1,141 @@
-Selamat Datang
---------------
+<!-- wiki -->
+[Beatmaps wikilink]: /wiki/Beatmaps/ "Beatmaps"
+[Game_Modifiers wikilink]: /wiki/Game_Modifiers/ "Game Modifiers"
+[Multi wikilink]: /wiki/Multiplayer/ "Multi-play"
+[Performance_Points wikilink]: /wiki/Performance_Points/ "Performance Points"
+[Performance_Points#FAQ wikilink]: /wiki/Performance_Points#faq "Performance Points FAQ"
+[Accuracy wikilink]: /wiki/Accuracy/ "Accuracy"
+[Achievements wikilink]: /wiki/Achievements/ "Achievements"
+[Chat_Console wikilink]: /wiki/Chat_Console/ "Chat Console"
+[FAQ wikilink]: /wiki/FAQ/ "FAQ"
+[Glossary wikilink]: /wiki/Glossary/ "Glossary"
+[osu!academy wikilink]: /wiki/Announcements/osu!academy/ "osu!academy"
+[Beginner wikilink]: /wiki/Beginner/ "Beginner"
+[Beatmapping wikilink]: /wiki/Beatmapping/ "Beatmapping"
+
+<!-- external -->
+[osu!tutorial_basic]: https://osu.ppy.sh/s/3756 "Installer's osu!tutorial by peppy"
+[Beatmaplist link]: https://osu.ppy.sh/p/beatmaplist "Official beatmaplist"
+[Packlist link]: https://osu.ppy.sh/p/packlist "Official packlist"
+[Beatmapping/Modding: Guide Compendium link]: https://osu.ppy.sh/forum/t/441929 "Beatmapping/Modding: Guide Compendium"
+
+<!-- image -->
+[osu-boxart image]: ./osu-boxart.png "osu!boxart"
+[welcome00 image]: ./Welcome00.png "Beatmap listing search"
+[welcome01 image]: ./Welcome01.png "Beatmap listing search results"
+[welcome02 image]: ./Welcome02.png "Beatmap download"
+[welcome03 image]: ./Welcome03.png "Opening the .osz file"
+[welcome04 image]: ./Welcome04.png "Beatmap in the Song Selection screen"
+[Chat_Console friendlist image]: ./Chat_Console_friendlist.png "Chat Console's friendlist"
+
+<!-- shared image -->
+[.osz icon link]: /wiki/shared/osz_icon_full.png ".osz file, clean beatmap file"
+[Accuracy results image]: /wiki/shared/Accuracy_results.png "Accuracy of player"
+
+<!-- Title -->
+# Selamat Datang
+![osu!][osu-boxart image]
 
 Pertama-tama kami ingin memberikan sambutan dan pelukan hangat karena kamu telah bergabung dengan komunitas kami. Biar kami jelaskan sebagian hal yang cepat sebelum kamu memulai perjalananmu disini.
 
-![osu!](Osu-boxart.png "osu!")
-
-\_\_TOC\_\_
-
-Tutorial
---------
-
+# Tutorial
 Sebelum kamu mulai bermain osu!, kamu dianjurkan untuk memainkan beatmap tutorial pertama ([Peter Lambert - osu! tutorial](http://osu.ppy.sh/s/3756)).
```

Comment:
for all links with `http://osu.ppy.sh/` please use `https://osu.ppy.sh/` (use the `https`, protocol)